packages/react-composites/src/composites/common/SendBox.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { useMemo } from 'react';
+import { SendBoxStylesProps, SendBox as SimpleSendBox } from '@internal/react-components';
+import { usePropsFor } from '../ChatComposite/hooks/usePropsFor';
+import { ChatCompositeOptions } from '../ChatComposite';
+/* @conditional-compile-remove(rich-text-editor) */
+// import { Suspense } from 'react';
```

Comment:
Do we still need this?

Reply:
Oh nice catch! That got lost when reverting some of the other changes.

Will fix now!